### PR TITLE
Added missing dependencies

### PR DESCRIPTION
**Pull request by core23**
### Changelog

```markdown
### Added
- Added `SonataAdminBundle` to suggest 
- Added `SonataMediaBundle` to suggest 
```

### Subject

The ``SonataAdminBundle`` and ``SonataMediaBundle`` should be suggested, because there is an integration in the admin backend.

### To do

- [X] Add missing dependencies

### CODE REVIEW

**Comment by OskarStark** [comment]
LGTM, but travis failed

**Comment by jordisala1991** [comment]
Versions should be ^3.0 I think

**Comment by core23** [comment]
There are no stable versions yet

**Comment by Soullivaneuh** [comment]
> There are no stable versions yet

@core23 admin-bundle has a stable version from time: https://packagist.org/packages/sonata-project/admin-bundle

And in any way, we should use 3.x.

For media-bundle, does the project have test requiring this dep? If not, it should be removed from `require-dev` for the moment.

**Comment by core23** [comment]
I would add missing tests afterwards

**Comment by Soullivaneuh** [comment]
> I would add missing tests afterwards

Same thing for `sonata-project/admin-bundle`?

In this case, could you add this dependency on another PR with test? In this case, we can merge this one first with the `suggest` section.

**Comment by core23** [comment]
Moved the stuff to `suggest` and `conflict`.

File: composer.json [hunk 1]
```diff
@@ -31,6 +31,14 @@
         "symfony/validator": "^2.3 || ^3.0",
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
+    "suggest": {
+        "sonata-project/admin-bundle": "For using the admin media browser.",
+        "sonata-project/media-bundle": "For using the admin media browser."
+    },
+    "conflict": {
+        "sonata-project/admin-bundle": "<3.0",
```
Comment (Soullivaneuh) [inline composer.json]: ` || >=4.0`. Same for line below. :+1:

Comment (core23) [inline composer.json]: Good point 👍

**Comment by core23** [comment]
> For media-bundle, does the project have test requiring this dep? If not, it should be removed from require-dev for the moment.

Sure we should use `require-dev` only for running tests @Soullivaneuh ? Composer [docs](https://getcomposer.org/doc/04-schema.md#require-dev) say, that you could use them for development too. 

For example when running code inspection, I got "class not found errors" because it can't resolve the admin classes.

**Pull request merged**